roles/postgres-cluster-pgbouncer-client-setup/templates/userlist.txt.j2

```diff
@@ -1,4 +1,4 @@
 "{{ postgresql_cluster_pg_bouncer_root_user | default('pgbounceradmin') }}" "{{ postgresql_cluster_pg_bouncer_root_password_hashed | default('md5' + ((postgresql_cluster_pg_bouncer_root_password + (postgresql_cluster_pg_bouncer_root_user | default('pgbounceradmin'))) | hash('md5'))) }}"
-{% for user in postgresql_cluster_users | default([]) %}
-"{{ user.name }}" "{{ user.hashed_password | default('md5' + ((user.password + user.name) | hash('md5'))) }}"
-{% endfor %}
\ No newline at end of file
+{% for user in passsword_hashes.results | default([]) %}
```

Comment:
i see we remove support for md5 altogether. Should we do that?

Reply:
It is recommended to remove MD5 support completely.  MD5 support is disabled by default in Postgres 14 and above. It is highly insecure and IMHO poor practice to enable it as the Postgres team specifically disables it by default.

Reply:
okay. understood. Then we should get rid of it completely.